.scalafmt.conf

```diff
@@ -0,0 +1,47 @@
+style = defaultWithAlign
```

Comment:
when I updated the other plugins we started seeing moans about this file being missing